### PR TITLE
Get rid of @ExperimentalUnsignedTypes on our declarations that levera…

### PR DESCRIPTION
**Pull request by qwwdfsad**
…ge non-experimental unsigned types

### CODE REVIEW

**Comment by sandwwraith** [comment]
I think we can remove experimentality, I don't remember any problems with them

**Comment by qwwdfsad** [comment]
I'll do it in a separate PR then

File: formats/json/commonMain/src/kotlinx/serialization/json/internal/StreamingJsonDecoder.kt [hunk 1]
```diff
@@ -16,7 +16,7 @@ import kotlin.jvm.*
 /**
  * [JsonDecoder] which reads given JSON from [AbstractJsonLexer] field by field.
  */
-@OptIn(ExperimentalSerializationApi::class, ExperimentalUnsignedTypes::class)
+@OptIn(ExperimentalSerializationApi::class)
```
Comment (sandwwraith) [inline formats/json/commonMain/src/kotlinx/serialization/json/internal/StreamingJsonDecoder.kt:19]: You've missed `StreamingJsonEncoder`

**Review by sandwwraith — CHANGES_REQUESTED** [review]
We also need to change a bit of documentation here: https://github.com/Kotlin/kotlinx.serialization/blob/8be6845927414a844ffebedc871dd8bc0d4b8aee/docs/builtin-classes.md#L71

**Review by sandwwraith — APPROVED** [review]

**Pull request merged**